core-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -1941,7 +1941,7 @@
         <source>Backup codes can be used to access your account if you have lost your authentication device and cannot generate two-factor codes anymore. Treat your backup codes with the same level of attention as your password! It is recommended to save them with a password manager such as Lastpass, 1Password or Keeper.</source>
       </trans-unit>
       <trans-unit id="MSC.twoFactorBackupCodesInfo">
-        <source>Put the codes in a safe spot. If you lose your authentication device and do not have backup codes, you cannot access your account anymore!</source>
+        <source>Make sure to copy the codes now. You will not be able to see them again!</source>
```

Comment:
The original hint still seems valid to me, do we really need to remove it?

Reply:
The hint is already in the description above, so it makes sense to remove it here and use the info message to point out that the codes are only visible once.